app/components/filters_component/filters_component.js

```diff
@@ -19,99 +18,90 @@ window.addEventListener(
 );
 
 export const init = (removeButtonSelector, clearButtonId, closeButtonId, showFilterPanelId) => {
+  const showFilterPanelEl = document.getElementById(showFilterPanelId);
+  const closeFilterPanelEl = document.getElementById(closeButtonId);
+  const clearFiltersEl = document.getElementById(clearButtonId);
+
```

Comment:
Perhaps not necessary, but maybe it's a bit easier to read if they're in the same order that they're in when passed into the init function?